modules/web/src/app/cluster/list/external-cluster/template.html

```diff
@@ -14,33 +14,34 @@
 limitations under the License.
 -->
 
```

Comment:
This comment is for **External Cluster Details** page:


I think layout what you have followed in **Cluster** details page must be followed for **External** details page

**see**
- Move top buttons for details page just like did in cluster details page wdyt? `...` 
- Add Machine Deployment
- Quota Details widgets

![missing-in-external-clusters](https://user-images.githubusercontent.com/17727069/204302418-39f8e9bd-0753-41d0-906e-683146f5d31e.png)

Reply:
UI is updated 
<img width="1467" alt="image" src="https://user-images.githubusercontent.com/56511988/204365051-2ffd0ad8-3407-498d-bba3-317ce7764480.png">